src/variety/haisu.js

```diff
@@ -148,6 +148,25 @@
 		}
 	},
 
+	Border:{
+		setLine: function(id) {
+			this.setLineVal(1);
+			if (this.qsub === 2) {
+				this.setQsub(0);
+			}
+		},
+		removeLine: function(id) {
+			this.setLineVal(0);
+			if (this.qsub === 2) {
+				this.setQsub(0);
+			}
+		},
+		removePeke: function(id) {
+			this.setLineVal(0);
+			this.setQsub(0);
+		}		
+	},
```

Comment:
This code is copied from `nagare.js`. Maybe this can be moved to common and be the default behavior? I don't immediately see a reason to ever erase qsub when drawing a line, other than when `qsub===2` (peke)

Reply:
I tend to agree. Willing to risk the change.

Reply:
I made the change, and deleted the `removePeke()` that was introduced in https://github.com/robx/pzprjs/commit/74eba4221965f3663fbd3f302b3d049b671e3ef9

Reply:
Actually it feels weird that other qsubs don't get removed when dragging in "erase peke" mode, so I re-added an override (the other way from the linked commit)